docs/Developers-Guide.md

```diff
@@ -203,9 +203,10 @@ intelmq/
     system.conf
 ```
 
-Assuming you want to create a bot for 'Abuse.ch Zeus' feed. It turns out that here it is necessary to create different parsers for the respective kind of events (C&C, Binaries, Dropzones). Therefore, the hierarchy ‘intelmq/bots/parser/abusech/parser.py’ would not be suitable because it is necessary to have more parsers, as mentioned above. The solution is to use the same hierarchy with an additional "description" in the file name, separated by underscore. Also see the section *Directories and Files naming*.
+Scenario:
+Assuming that you want to create a bot for 'Abuse.ch Zeus' feed. It turns out, just as an example, that here it is necessary to create different parsers for the respective kind of events (C&C, Binaries, Dropzones). Therefore, the hierarchy ‘intelmq/bots/parser/abusech/parser.py’ would not be suitable because it is necessary to have more parsers, as mentioned above. The solution is to use the same hierarchy with an additional "description" in the file name, separated by underscore. Also see the section *Directories and Files naming*.
 
-Example:
+Example: (the following filepaths does NOT exist, they are just for learning purposes)
```

Comment:
Why don't you replace the old examples with current ones? E.g. intelmq/bots/parsers/abusech/parser*.py